src/config.rs

```diff
@@ -2262,8 +2262,8 @@ impl Default for CdcConfig {
     fn default() -> Self {
         Self {
             min_ts_interval: ReadableDuration::secs(1),
-            old_value_cache_size: 1024,
-            hibernate_regions_compatible: true,
```

Comment:
@5kbpers Is it stable? Is it enabled in release-5.0 by default?

Reply:
Yes, a few weeks ago and last week I tested it with tipocket and some manual tests, did not find any other problem except rolling update.

Reply:
Agree to disable it first, we have some other problems that need to be fixed.

Reply:
Better to enable it in master and disable it in 5.0